Makefile

```diff
@@ -3,7 +3,7 @@ NAME        := terraform-docs
 VENDOR      := segmentio
 DESCRIPTION := Generate docs from Terraform modules
 MAINTAINER  := Martin Etmajer <metmajer@getcloudnative.io>
-URL         := https://github.com/$(VENDOR)/$(NAME)
+URL         := https://github.com/segmentio/terraform-docs
```

Comment:
I don’t understand why you’re removing variable extrapolation now. This should be reverted, even if URL is currently unused.

Reply:
Sure.

Reply:
Thanks!